tests/testthat/test-utils.R

```diff
@@ -219,3 +219,36 @@ test_that("Confirms if threads = 1, parallel is not set [BBR-UTL-015]", {
   })
 })
 
+test_that("confirms handling of default parameters [BBR-UTL-016]", {
+  withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()),{
+    withr::with_tempdir({
```

Comment:
This looks like a good regression test.  I've confirmed that, if I revert the code changes, I get the failure from gh-553

I wonder whether we can simplify things a bit, making the core of the test stand out more.

Some notes:
  * `with_tempdir` puts you in a tempdir, so there is no need to create more tempdirs, use absolute paths, or clean up the directory afterword.
  * some of the model setup is unneeded for this test
  * if it's easy to avoid a warning, do so rather than suppressing it

I think this test could be reduced:

```R
test_that("confirms handling of default parameters [BBR-UTL-016]", {
  withr::with_tempdir({
    fs::file_copy(file.path(ABS_MODEL_DIR, "1.ctl"), ".")
    cat("", file = "bbi.yaml")
    mod1 <- new_model("1", .bbi_args = list(parallel = FALSE))
    res <- submit_model(mod1, .dry_run = TRUE)
    expect_false(all(str_detect(res$cmd_args, "--parallel")))
  })
})
```

Note that that triggers another bug in utils.R.  You could avoid it by adding another arg to `.bbi_args`, but I think it'd be better to fix it with something like this:

```diff
diff --git a/R/utils.R b/R/utils.R
index b858eacf..8fd8c86c 100644
--- a/R/utils.R
+++ b/R/utils.R
@@ -113,7 +113,13 @@ format_cmd_args <- function(.args, .collapse = FALSE) {
   if (.collapse) {
     return(paste0(arg_vec, collapse = " "))
   } else {
-    return(arg_vec %>% set_names(NULL))
+    res <- if (length(arg_vec) > 0) {
+      arg_vec %>% set_names(NULL)
+    } else {
+      arg_vec
+    }
+
+    return(res)
   }
 }
 
```